src/main/java/at/ac/tuwien/kr/alpha/common/predicates/TotalOrder.java

```diff
@@ -0,0 +1,85 @@
+package at.ac.tuwien.kr.alpha.common.predicates;
+
+import at.ac.tuwien.kr.alpha.common.terms.Term;
+import at.ac.tuwien.kr.alpha.grounder.Substitution;
+
+import java.util.List;
+
+public class TotalOrder implements Evaluable, Predicate {
```

Comment:
TotalOrder is more general than what this actually is (it is a certain instance of a total order). Since this boils down to the builtin atom, I suggest calling it similar to that.

Reply:
OK, so `BuiltinOrder`.